### ISSUE TITLE
billing email for SaaS

### ISSUE THREAD

**Issue by homeyer** [waffleio/waffle.io#2841]
Stripe will send email receipts, if we configure it to do so. We currently don't. https://support.stripe.com/questions/email-receipts

AC:
- [x] paying users get an email receipt from stripe when a charge goes through
  - ~~looks like setting `receipt_email` as part of the charge creation request will do the trick~~ nope, stripe customer needs an email tacked on

Out of scope: configuring in the waffle ui what email address to use, let's wait on a customer request before doing that.

**Comment by songawee** [comment]
We do get the email the user puts in, but we don't do anything with it currently - https://stripe.com/docs/checkout#integration-simple-options is how it is available

**Comment by burtonjc** [comment]
per @jtslear: can we use intercom for this?

**Comment by songawee** [comment]
- [ ] add info on help page for how and why emails work

**Comment by kellihsf** [comment]
link to draft: https://app.intercom.io/a/apps/fb774a034de340cde33822ae0c071ad60efe899f/educate/articles/414250/show
